### ISSUE TITLE
v10 Compatibility

### ISSUE THREAD

**Issue by TRexStark** [sotirisf/Umbraco-AutoNode#14]
Hi There, any possibility to get this working with v10? on v10.2.0 it returns and error

One or more errors occurred. (Method not found: 'Umbraco.Cms.Core.Services.OperationResult Umbraco.Cms.Core.Services.IContentService.Save(Umbraco.Cms.Core.Models.IContent, Int32)'.)

**Comment by sotirisf** [comment]
Yes, I'm planning to have a release for v10 within the month if workload permits.
